utils/copy-button-transformer.js

```diff
@@ -0,0 +1,114 @@
+
+const whitespaceRegEx = /\s*\n\s*/g;
+const trimWhitespace = (input) => input.replaceAll(whitespaceRegEx, "").trim();
+
+
+function transformerCopyButton(options = {
+    feedbackDuration: 3000
+}) {
+    return {
+        name: "copy-button-transformer",
+        code(node) {
+            node.children.unshift({
+                type: "element",
+                tagName: "button",
+                properties: {
+                    type: "button",
+                    title: "Copy code",
+                    "aria-label": "Copy code",
```

Comment:
```suggestion
                    "aria-label": "Copy",
```

Reply:
Can you add the title and the aria-lable to successIcon as `Copied` to match with others.

Reply:
Done